getting-started/vs-integration/new-project-wizard.md

```diff
@@ -50,31 +52,28 @@ To start the wizard, use either of the following options:
 
 ## Using the Wizard
 
-In the order of their appearance, the Create New Project wizard presents the following screens with options that will assist you to configure your new app:
-
-1. On the first screen, you can select a location and a name for your project.
+In the order of their appearance, the Create New Project wizard presents several screens with options that will assist you to configure your new app.
 
-2. On the second screen, you can choose:
+After you select the **Create New Project** option from the extensions menu, Visual Studio will open a **Configure your new project** window where you can select a location and a name for your project. 
```

Comment:
Switching to present simple works equally well here. It's our [recommended tense](https://docs.telerik.com/style-guide/cheat-sheet#use-simple-tenses). 
```suggestion
After you select the **Create New Project** option from the extensions menu, Visual Studio opens a **Configure your new project** window where you can set the location and the name for your project. 
```